### DISCUSSION TITLE
Using the RecycleView library

### DISCUSSION THREAD

**Discussion by JohnCTXP** [microsoft/vscode-discussions#2329] [Extension Development QnA]
Can anyone provide me examples on how to use the RecycleView library widget in a Python source code file and in a Kivy script file?

**Comment by gjsjohnmurray** [comment]
Are you creating a VS Code extension?

**Reply by JohnCTXP** [reply]
Yes

**Reply by JohnCTXP** [reply]
I already have Kivy Python installed on my Visual Studio Code for my desktop or laptop

**Reply by burkeholland** [reply]
@JohnCTXP is this still an issue? Were you able to figure out how to get this to work?

**Comment by olguzzar** [comment]
Going to close this discussion as there's been a period of inactivity. If you have more information to add, please let us know and we can re-open!
